src/documentation.jl

```diff
@@ -255,18 +342,179 @@ See also: [`CryptoContext`](@ref), [`Ciphertext`](@ref)
+
+See also: [`CryptoContext`](@ref), [`Ciphertext`](@ref)
+"""
+EvalSin
+
+"""
+    EvalCos(crypto_context::CryptoContext, ciphertext::Ciphertext, a::Float64, b::Float64,
+            degree::UInt32)
+
+Evaluate approximate cosine function on a given `ciphertext` using the Chebyshev approximation over
+the range ``[a,b]``. Return the resulting [`Ciphertext`](@ref). The input ciphertext needs to be
+derived from the given `crypto_context`.
+
+`a` is a lower bound of elements contained in the given `ciphertext`.
+
+`b` is an upper bound of elements contained in the given `ciphertext`.
+
+`degree` is a desired degree of approximation.
+
+Supported only in CKKS.
+
+See also: [`CryptoContext`](@ref), [`Ciphertext`](@ref)
+"""
+EvalCos
+
+"""
+    EvalLogistic(crypto_context::CryptoContext, ciphertext::Ciphertext, a::Float64, b::Float64,
+                 degree::UInt32)
+
+Evaluate approximate logistic function ``\\frac{1}{1 + \\exp{-x}}`` on a given `ciphertext` using
+the Chebyshev approximation over the range ``[a,b]``. Return the resulting [`Ciphertext`](@ref).
+The input ciphertext needs to be derived from the given `crypto_context`.
+
+`x` is an element contained in the given `ciphertext`
+                    
+`a` is a lower bound of elements contained in the given `ciphertext`.
+                    
+`b` is an upper bound of elements contained in the given `ciphertext`.
+                    
+`degree` is a desired degree of approximation.
+                    
+Supported only in CKKS.
+                    
+See also: [`CryptoContext`](@ref), [`Ciphertext`](@ref)             
+"""
+EvalLogistic
+
+"""
+    EvalDivide(crypto_context::CryptoContext, ciphertext::Ciphertext, a::Float64, b::Float64,
+               degree::UInt32)
+
+Evaluate approximate division function ``\\frac{1}{x}`` where ``x >= 1`` on a given `ciphertext`
+using the Chebyshev approximation over the range ``[a,b]``. Return the resulting [`Ciphertext`](@ref).
+The input ciphertext needs to be derived from the given `crypto_context`.
+
+`x` is an element contained in the given `ciphertext`
+                    
+`a` is a lower bound of elements contained in the given `ciphertext`.
+                    
+`b` is an upper bound of elements contained in the given `ciphertext`.
+                    
+`degree` is a desired degree of approximation.
+                    
+Supported only in CKKS.
+                    
+See also: [`CryptoContext`](@ref), [`Ciphertext`](@ref)             
+"""
+EvalDivide
+
+"""
+    EvalSumKeyGen(crypto_context::CryptoContext, private_key::PrivateKey, public_key::PublicKey = nothing)
+
+Generates the key map to be used by [`EvalSum`](@ref). `public_key` has to be set for NTRU schemes.
+
+Please refer to the OpenFHE documentation for more details.
+
+See also: [`CryptoContext`](@ref), [`PrivateKey`](@ref), [`PublicKey`](@ref), [`EvalSum`](@ref)
+"""
+EvalSumKeyGen
+
+"""
+    EvalSum(crypto_context::CryptoContext, ciphertext::Ciphertext, capacity::UInt32)
```

Comment:
```suggestion
    EvalSum(crypto_context::CryptoContext, ciphertext::Ciphertext, batch_size::Integer)
```
Important: Here in OpenFHE.jl we do _not_ try to be "smarter" (or easier to use) than OpenFHE itself. Here, the convenience for the user comes from us trying to stick as close as possible to the original source. Thus I'd prefer to call it `batch_size` (since that's the parameter also used in OpenFHE).